seg_utils/src/label_main.py

```diff
@@ -2,7 +2,7 @@
 
 from PyQt5.QtCore import pyqtSignal, QPointF, QRectF, Qt, QSize
 from PyQt5.QtWidgets import QMainWindow, QFileDialog, QListWidgetItem, QMessageBox, QMenu
-from PyQt5.QtGui import QPixmap, QIcon
+from PyQt5.QtGui import QPixmap, QIcon, QCloseEvent
```

Comment:
I think we can change PyQt imports to be more general since that is the standard with this library:
```python
from PyQt5.QtCore import *
...
Generally, not how I like to do things, but it can get too messy with Qt when not doing that.
```

Reply:
Also, we should move to PyQt6, because PyQt5 isn't supported on the new macs (yet, at least) and there's no reason not to...